Docs/SPE 260: update and reorganize telemetry metric references (#21965)
hashicorp/vault: website/content/partials/telemetry-metrics/vault/raft_storage/follower/last_heartbeat_ms.mdx

### vault.raft_storage.follower.last_heartbeat_ms ((#vault-raft_storage-follower-last_heartbeat_ms))

Metric type | Value | Description
----------- | ----- | -----------

> gauge       | ms    | Time since the follower last received an echo request

Reviewer: ```suggestion
gauge       | ms    | Time since the follower last received a heartbeat request
```
